portal/src/texts/core/motion.md

```diff
@@ -1,3 +1,15 @@
 ## Bevegelse
 
-I Fremtind ønsker vi å vise at vi er et selskap i bevegelse. Derfor bruker vi animasjon for å hjelpe brukeren med å forstå hva som skjer når de bruker løsningene våre. Vi benytter det sparsomt men godt – det vil si at vi animerer bare når brukeren utfører en handling i en av løsningene våre. Animasjoner skal alltid ha en funksjonell verdi.
+For å sikre at Fremtind fremstår som et selskap i bevegelse med en tydelig retning fremover, er bevegelse definert som et av våre designprinsipper. I ordet ligger det at vi er utforskende, fremtidsrettet og tilpasningsdyktige.
+
+Vi leter stadig etter nye og bedre måter å løse problemer på, og er ikke redde for å utfordre dagens standard. Bruk av animasjon er med på å synliggjøre Fremtind som et selskap i bevegelse. **Animasjoner skal alltid ha en funksjonell verdi**.
+
+### Nøkkelord
+
+-   Alltid i bevegelse
+-   Tilpasningsdyktig
+-   Progressivt
+
+### I bruk
+
+Benytt systemet for [bevegelse]() i Jøkul som utgangspunkt når du skal skape animasjoner i løsningene du lager. Sørg for at all animasjon har en tydelig funksjon i å styre oppmerksomhet eller forventning. Det må likevel være mulig å bruke løsningene med redusert bevegelse for de som ønsker eller trenger det.
```

Comment:
Forslag: Når du skal skape animasjoner i en løsning du lager, bruker du de prinsippene og mulighetene Jøkul har for å skape bevegelse. Pass på at alle animasjoner får en tydelig funksjon og er med på å styre brukerens oppmerksomhet og forventning. Det skal likevel være mulig å bruke løsningene med redusert bevegelse, hvis noen heller ønsker eller trenger det.

Kommentar til forslaget: Jeg er litt usikker på hva "systemet for bevegelse" skal bety her, så dere får se om mitt forslag dekker det dere mener.

Reply:
Her er det meningen at det skal referere til et system for bevegelse og animasjoner som skal være en del av biblioteket. Det vil altså være et sett definerte bevegelseskurver og -varigheter. Nå er det litt usikkert om vi kommer til å ha med dette i første versjon, så forslaget ditt funker fint frem til det eventuel blir tatt med :)